docs/docs/guides/installation.mdx

```diff
@@ -151,11 +151,11 @@ Please reach out if you need some help and feel free to send a PR with the insta
 </TabItem>
 </Tabs>
 
-Once docker desktop and invoke are installed you can start and initialize the Infrahub demo environment with the following command:
+Once docker desktop and invoke are installed you can start and initialize Infrahub:
 
 ```shell
 cd infrahub
-invoke demo.start demo.load-infra-schema demo.load-infra-data
```

Comment:
Do we still need the Schema and Data loads though?

Reply:
This guide is more about getting Infrahub installed. Not about getting the demo environment setup, for that you would need to follow the getting started tutorial.

So I agree with @FragmentedPacket here. But maybe other people have a different opinion.

To avoid confusion we could maybe use a different namespace for this?